views/playlist/playlist.pug

```diff
@@ -1,4 +1,4 @@
-extends layout
+extends ../layout
```

Comment:
Do we need to change the playlist route files too?

Reply:
that was already included